packages/react-storage/package.json

```diff
@@ -46,7 +46,8 @@
     "tslib": "^2.5.2"
   },
   "peerDependencies": {
-    "aws-amplify": "^6.3.2",
+    "@aws-amplify/core": "^6.4.0",
```

Comment:
We're only using `@aws-amplify/core` in the storage package to reference a deprecated type (`StorageAccessLevel`), can we manually recreate the type and remove the peer dep? Can be done in a follow up

Reply:
Yeah will do this in a followup